crates/matrix-sdk-base/src/store/send_queue.rs

```diff
@@ -285,6 +298,17 @@ pub struct FinishUploadThumbnailInfo {
     pub height: Option<UInt>,
 }
 
+#[cfg(feature = "unstable-msc4274")]
+#[derive(Clone, Debug, Serialize, Deserialize)]
+/// Detailed record about a file and thumbnail used when finishing a gallery
+/// upload.
```

Comment:
1. Let's put the doc comment above the `cfg` and derive.
2. The name suggests this is for a single item (`finish gallery item`), but the code comment suggests it's when finishing a gallery upload (so I'd expect after all the items have been uploaded). Can you precise in the doc comment / name what's the truth?